tools/accuracy_checker/accuracy_checker/data_analyzer/base_data_analyzer.py

```diff
@@ -32,7 +32,12 @@ def resolve(cls, name):
 
     @staticmethod
     def object_count(annotations):
-        print_info('Total annotation objects: {size}'.format(size=len(annotations)))
+        annotations_size = len(annotations)
+        print_info('Total annotation objects: {size}'.format(size=annotations_size))
+        return annotations_size
+
 
     def analyze(self, result: list, meta, count_objects=True):
-        self.object_count(result)
+        data_analyze = {}
+        data_analyze['annotations_size'] = self.object_count(result)
+        return data_analyze
```

Comment:
```suggestion
        return {'annotation_size': self.object_count(result)}
```
BTW, what happens if count_objects=False will be passed to the method?